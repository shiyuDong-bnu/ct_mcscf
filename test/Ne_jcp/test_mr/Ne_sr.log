2025-05-09:18:45:37,106 INFO     [psi4.driver.task_planner:285] PLANNING Atomic:  keywords={'DF_BASIS_MP2': 'AUG-CC-PVDZ-OPTRI', 'D_CONVERGENCE': 1e-06, 'E_CONVERGENCE': 1e-10, 'FREEZE_CORE': '1', 'MAXITER': 40, 'SCF_TYPE': 'PK', 'SCREENING': 'CSAM', 'function_kwargs': {}}
2025-05-09:18:45:37,109 INFO     [psi4.driver.driver:523] Compute energy(): method=scf, basis=aug-cc-pvdz, molecule=default, nre=0.0
2025-05-09:18:45:37,392 INFO     [psi4.driver.driver:526] Return energy(): -128.49634973054066
