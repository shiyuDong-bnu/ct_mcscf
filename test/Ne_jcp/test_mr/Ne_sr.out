
Scratch directory: /data/home/sydong/scratch/psi4/

Error: option JOB_TYPE is not contained in the list of available options.

Did you mean? JOBTYPE SOS_TYPE

   => Libint2 <=

    Primary   basis highest AM E, G, H:  6, 6, 3
    Auxiliary basis highest AM E, G, H:  7, 7, 4
    Onebody   basis highest AM E, G, H:  -, -, -
    Solid Harmonics ordering:            Gaussian

*** tstart() called on hydrogen
*** at Fri May  9 18:45:37 2025

   => Loading Basis Set <=

    Name: AUG-CC-PVDZ
    Role: ORBITAL
    Keyword: BASIS
    atoms 1 entry NE         line   326 file /data/home/sydong/opt/psi4/share/psi4/basis/aug-cc-pvdz.gbs 


         ---------------------------------------------------------
                                   SCF
               by Justin Turney, Rob Parrish, Andy Simmonett
                          and Daniel G. A. Smith
                              RHF Reference
                        1 Threads,    500 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         NE           0.000000000000     0.000000000000     0.000000000000    19.992440176200

  Running in c1 symmetry.

  Rotational constants: A = ************  B = ************  C = ************ [cm^-1]
  Rotational constants: A = ************  B = ************  C = ************ [MHz]
  Nuclear repulsion =    0.000000000000000

  Charge       = 0
  Multiplicity = 1
  Electrons    = 10
  Nalpha       = 5
  Nbeta        = 5

  ==> Algorithm <==

  SCF Algorithm Type is PK.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-10
  Density threshold  = 1.00e-06
  Integral threshold = 1.00e-12

  ==> Primary Basis <==

  Basis Set: AUG-CC-PVDZ
    Blend: AUG-CC-PVDZ
    Number of shells: 9
    Number of basis functions: 23
    Number of Cartesian functions: 25
    Spherical Harmonics?: true
    Max angular momentum: 2

  ==> Integral Setup <==

  Using in-core PK algorithm.
   Calculation information:
      Number of atoms:                   1
      Number of AO shells:               9
      Number of primitives:             25
      Number of atomic orbitals:        25
      Number of basis functions:        23

      Integral cutoff                 1.00e-12
      Number of threads:                 1

  Performing in-core PK
  Using 76452 doubles for integral storage.
  We computed 1035 shell quartets total.
  Whereas there are 1035 unique shell quartets.

  ==> DiskJK: Disk-Based J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    Memory [MiB]:              375
    Schwarz Cutoff:          1E-12

    OpenMP threads:              1

  Minimum eigenvalue in the overlap matrix is 9.8616913390E-02.
  Reciprocal condition number of the overlap matrix is 4.4899065075E-02.
    Using symmetric orthogonalization.

  ==> Pre-Iterations <==

  SCF Guess: Core (One-Electron) Hamiltonian.

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A         23      23       5       5       5       0
   -------------------------------------------------------
    Total      23      23       5       5       5       0
   -------------------------------------------------------

  ==> Iterations <==

                        Total Energy        Delta E     RMS |[F,P]|

   @RHF iter   1:  -122.51940312077677   -1.22519e+02   1.52855e-01 ADIIS
   @RHF iter   2:  -119.86187195478323    2.65753e+00   1.69380e-01 ADIIS
   @RHF iter   3:  -128.44865676196707   -8.58678e+00   1.96201e-02 DIIS/ADIIS
   @RHF iter   4:  -128.49620248301107   -4.75457e-02   9.66785e-04 DIIS/ADIIS
   @RHF iter   5:  -128.49634482530493   -1.42342e-04   1.87240e-04 DIIS/ADIIS
   @RHF iter   6:  -128.49634968346354   -4.85816e-06   1.77153e-05 DIIS
   @RHF iter   7:  -128.49634972914518   -4.56816e-08   3.32025e-06 DIIS
   @RHF iter   8:  -128.49634973053509   -1.38991e-09   1.83211e-07 DIIS
   @RHF iter   9:  -128.49634973054066   -5.57066e-12   2.28381e-08 DIIS
  Energy and wave function converged.


  ==> Post-Iterations <==

    Orbital Energies [Eh]
    ---------------------

    Doubly Occupied:                                                      

       1A    -32.794951     2A     -1.940421     3A     -0.853040  
       4A     -0.853040     5A     -0.853040  

    Virtual:                                                              

       6A      0.287355     7A      0.293011     8A      0.293011  
       9A      0.293011    10A      1.757692    11A      1.757692  
      12A      1.757692    13A      1.757692    14A      1.757692  
      15A      1.961351    16A      1.961351    17A      1.961351  
      18A      2.741966    19A      6.372473    20A      6.372473  
      21A      6.372473    22A      6.372473    23A      6.372473  

    Final Occupation by Irrep:
              A 
    DOCC [     5 ]
    NA   [     5 ]
    NB   [     5 ]

  @RHF Final Energy:  -128.49634973054066

   => Energetics <=

    Nuclear Repulsion Energy =              0.0000000000000000
    One-Electron Energy =                -182.4037175925750205
    Two-Electron Energy =                  53.9073678620343628
    Total Energy =                       -128.4963497305406577

Computation Completed


Properties will be evaluated at   0.000000,   0.000000,   0.000000 [a0]

Properties computed using the SCF density matrix


 Multipole Moments:

 ------------------------------------------------------------------------------------
     Multipole            Electronic (a.u.)      Nuclear  (a.u.)        Total (a.u.)
 ------------------------------------------------------------------------------------

 L = 1.  Multiply by 2.5417464519 to convert [e a0] to [Debye]
 Dipole X            :         -0.0000000            0.0000000           -0.0000000
 Dipole Y            :         -0.0000000            0.0000000           -0.0000000
 Dipole Z            :          0.0000000            0.0000000            0.0000000
 Magnitude           :                                                    0.0000000

 ------------------------------------------------------------------------------------

*** tstop() called on hydrogen at Fri May  9 18:45:37 2025
Module time:
	user time   =       0.22 seconds =       0.00 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
Total time:
	user time   =       0.22 seconds =       0.00 minutes
	system time =       0.05 seconds =       0.00 minutes
	total time  =          0 seconds =       0.00 minutes
    Molecular point group: c1
    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
         NE           0.000000000000     0.000000000000     0.000000000000    19.992440176200

  -AO BASIS SET INFORMATION:
    Name                   = AUG-CC-PVDZ
    Blend                  = AUG-CC-PVDZ
    Total number of shells = 9
    Number of primitives   = 25
    Number of AO           = 25
    Number of SO           = 23
    Maximum AM             = 2
    Spherical Harmonics    = TRUE

  -Contraction Scheme:
    Atom   Type   All Primitives // Shells:
   ------ ------ --------------------------
       1    NE     18s 5p 2d // 4s 3p 2d 

  ==> AO Basis Functions <==

    [ AUG-CC-PVDZ ]
    spherical
    ****
   NE   1
    S   8 1.00
                     17880.00000000           0.00073800
                      2683.00000000           0.00567700
                       611.50000000           0.02888300
                       173.50000000           0.10854000
                        56.64000000           0.29090700
                        20.42000000           0.44832400
                         7.81000000           0.25802600
                         1.65300000           0.01506300
    S   8 1.00
                     17880.00000000          -0.00017200
                      2683.00000000          -0.00135700
                       611.50000000          -0.00673700
                       173.50000000          -0.02766300
                        56.64000000          -0.07620800
                        20.42000000          -0.17522700
                         7.81000000          -0.10703800
                         1.65300000           0.56705000
    S   1 1.00
                         0.48690000           1.00000000
    P   3 1.00
                        28.39000000           0.04608700
                         6.27000000           0.24018100
                         1.69500000           0.50874400
    P   1 1.00
                         0.43170000           1.00000000
    D   1 1.00
                         2.20200000           1.00000000
    S   1 1.00
                         0.12300000           1.00000000
    P   1 1.00
                         0.10640000           1.00000000
    D   1 1.00
                         0.63100000           1.00000000
    ****

    Orthogonalizing basis for space RIBS.
    0 linear dependencies will be 'removed'.
    Projecting out 'ALL' from 'RIBS' to obtain space 'CABS'
    0 linear dependencies will be 'removed'.
        Orbital space before projecting out:    (n = 1): 92  
        Orbital space after projecting out:     (n = 1): 69  

*********************************************************************************************************************
When computing potential integrals with different bra and ket basis, the atom definition is taken from the bra basis.
*********************************************************************************************************************
*********************************************************************************************************************
When computing potential integrals with different bra and ket basis, the atom definition is taken from the bra basis.
*********************************************************************************************************************
